src/styles/main.scss

```
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-template-rows: 300px 300px;
+
+  &__item {
+    display: inline-grid;
+    place-items: center;
+
+    min-height: 300px;
```

```suggestion
    height: 300px;
```
simple height is also ok here